src/server/qgsconfigcache.h

```diff
@@ -65,21 +105,155 @@ class SERVER_EXPORT QgsConfigCache : public QObject
      */
     const QgsProject *project( const QString &path, const QgsServerSettings *settings = nullptr );
 
-  private:
-    QgsConfigCache() SIP_FORCE;
+    /**
+     * Set the invalidation strategy for the cache
+     * The cache takes ownership of the strategy which will be deleted on cleanup
```

Comment:
What does "on cleanup" mean? Remove this or clarify.

Reply:
Fixed.